src/page-renderers/Components/Header.jsx

```diff
@@ -1,26 +1,46 @@
 const React = require('react')
+const getHeaderLinks = require('../../helpers/get-header-links')
 
 const Styles = {
   navbar: {
     border: 'none',
     borderRadius: '0',
     backgroundColor: '#000',
-    minHeight: '60px',
+    height: '60px',
     marginBottom: '0',
     width: '100%',
     paddingRight: '5px',
     paddingLeft: '5px'
+  },
+  headerLinks: {
+    marginTop: '5px'
   }
 }
 
-const Header = () => (
-  <div className='container' style={Styles.navbar}>
-    <div className='row col-md-12' style={{ width: '100%', paddingTop: '10px', paddingRight: '0px', paddingLeft: '25px' }}>
-      <h3 className='col-md-4' style={{ textAlign: 'left', paddingRight: '0px', paddingBottom: '20px', color: '#fff' }}>
-        Morty-Docs
-      </h3>
+const Header = ({ relPath }) => {
+  let headerLinks
+  let pathParts
+
+  if (relPath) {
+    pathParts = relPath.split('/')
+    headerLinks = getHeaderLinks(relPath, pathParts)
+  }
+
+  return (
+    <div className='container' style={Styles.navbar}>
```

Comment:
Was just talking about accessibility on this with peeps. We probably want a `nav` > `ul` > `li` > `a` on these.

Reply:
For context: screen readers can parse `nav` elements as navigation elements